lib/player.js

```diff
@@ -62,7 +62,8 @@ Player.prototype = {
       if((distance < this.mass * 0.8) &&
          (players[i] !== this) &&
          (this.mass * 0.8 > players[i].mass)) {
```

Comment:
.8 threshold is too high. Once circles get larger there are times when one should definitely win but can't.